src/containers/EnterpriseApp/EnterpriseApp.test.jsx

```diff
@@ -105,6 +115,20 @@ describe('<EnterpriseApp />', () => {
     expect(wrapper.text()).toContain(404);
   });
 
+  it('renders the load page correctly', () => {
+    const store = mockStore({
+      ...initialState,
+      portalConfiguration: {
+        ...portalConfigurationBeforeLoad,
```

Comment:
not: since loading is the only variable of interest, is it a little bit tighter of a test to just change `loading` in the overrride, e.g. , 

```
portalConfiguration: { 
   { ...initialState.portalConfiguration, loading: true },
}
```

just to ensure that this test only focuses on loading as a state change.

Reply:
oooh good idea. 👍